Header header

string[] arm_name
geometry_msgs/Point goal_position
geometry_msgs/Quaternion goal_orientation
